### ISSUE TITLE
Some commands dont return multiple values

### ISSUE THREAD

**Issue by pmackay** [maxlath/wikidata-cli#41]
I've tried `summary`, `label` and `description` all of which suggest it is possible to get multiple results by providing multiple IDs, e.g.:

`wd label Q18120925 Q22117436 Q22117437`

I can only get the first result back. Am I doing anything wrong?

**Comment by maxlath** [comment]
this feature is available only since 5 days, when the `v5.0.0` was published, could it be that your version is older than that?

**Comment by pmackay** [comment]
I do apologise, I should have checked for having the latest version first. I updated and can get it working. However I did have to also update `node` to the latest, I had `v6.10.0` before which is later than the version recommended for this, but on that version I got this error:

```
TypeError: Object.values is not a function
    at entities (/Users/pmackay/.config/yarn/global/node_modules/wikidata-cli/lib/entity_attribute_command.js:14:33)
    at process._tickCallback (internal/process/next_tick.js:103:7)
Context: undefined
```

**Comment by maxlath** [comment]
no problem :)
thanks for spotting the `Object.values` issue, it indeed appears to only be available from Node v7.0, so I'm replacing those to avoid having to ask everyone to update Node

**Comment by maxlath** [comment]
all occurrences of `Object.values` got removed in ee5e90c, the updated version was published as `v5.0.2`
